test/kata5.reachDestination.test.js

```diff
@@ -1,5 +1,7 @@
 const { reachDestination } = require("../src");
 
 describe("reachDestination", () => {
-  test("returns string with estimated time of arrival", () => {});
+  test("returns string with estimated time of arrival", () => {
```

Comment:
Consider whether it is a good idea to add more assertions to a test for extra assurance. 

For example, I could rewrite the function to always return "I will be there in 4.5 hours" and this test would pass but it would not have the intended behaviour. 

I know that seems a bit pedantic in this case but the basic premise is still valid. By adding at least one more assertion, you have made the test more robust.